### ISSUE TITLE
chef 14 error: cannot load such file -- chef/mixin/language

### ISSUE THREAD

**Issue by Stromweld** [Microsoft/vsts-agent-cookbook#28]
Getting error loading vsts_agent cookbook on chef-client 14. This line seems to break it: https://github.com/Microsoft/vsts-agent-cookbook/blob/master/libraries/service.rb#L4

```
Starting Chef Client, version 14.3.37
resolving cookbooks for run list: ["pl_roundhouse_deployment_role"]
Synchronizing Cookbooks:
  - bash-completion (0.1.0)
  - dstat (0.1.1)
  - logrotate (2.2.0)
  - nano (1.0.3)
  - vim (2.0.2)
  - trusted_certificate (3.2.0)
  - chef-sugar (4.0.0)
  - timezone_iii (1.1.6)
  - sshd (1.3.1)
  - build-essential (8.1.1)
  - pl_katello_client (1.0.5)
  - redhat_subscription_manager (1.0.4)
  - selinux_policy (2.1.0)
  - powershell (6.1.2)
  - windows (4.2.5)
  - line (1.0.6)
  - nfs (2.5.1)
  - sudo (5.4.0)
  - sysctl (1.0.5)
  - systemd (3.2.3)
  - seven_zip (3.0.0)
  - ohai (5.2.3)
  - selinux (2.1.1)
  - chef_hostname (1.0.1)
  - mingw (2.1.0)
  - ms_dotnet (4.2.0)
  - vsts_agent (2.0.3)
  - pl_active_directory (1.2.1)
  - pl_install_certificates (1.0.0)
  - pl_tfs_agent (1.1.0)
  - pl_roundhouse_deployment_role (1.0.0)
  - audit (7.0.1)
  - ark (4.0.0)
  - chef-client (11.0.0)
  - chef_client_updater (3.4.0)
  - cron (6.2.0)
  - firewall (2.6.5)
  - ntp (3.6.0)
  - postfix (5.3.1)
  - pl_active_directory_wrap (1.2.1)
  - pl_base_role (1.3.5)
Installing Cookbook Gems:
Compiling Cookbooks...

================================================================================
Recipe Compile Error in C:/chef/cache/cookbooks/vsts_agent/libraries/service.rb
================================================================================

LoadError
---------
cannot load such file -- chef/mixin/language

System Info:
------------
chef_version=14.3.37
platform=windows
platform_version=10.0.14393
ruby=ruby 2.5.1p57 (2018-03-29 revision 63029) [x64-mingw32]
program_name=C:/opscode/chef/bin/chef-client
executable=C:/opscode/chef/bin/chef-client


Running handlers:
Running handlers complete
Chef Client failed. 0 resources updated in 13 seconds
```

**Comment by Stromweld** [comment]
further digging it looks like chef/mixin/language was depracated and it is recommended to use one or both of: `Chef::DSL::PlatformIntrospection or Chef::DSL::DataQuery`
https://github.com/chef/chef/blob/522d04e230cb0d5d8b7878baed568d069ea2d8c0/lib/chef/mixin/language.rb#L45

**Comment by ivadim** [comment]
Changes were merged and released in v2.0.4

Thank you for the contribution.
